src/applications/vaos/components/ListItem.jsx

```diff
@@ -10,7 +10,7 @@ export default function ListItem({
   status,
 }) {
   const uniqueIdentifier = appointment.id || appointment.uuid;
-  const idClickable = `id-${uniqueIdentifier.replace('.', '\\.')}`;
+  const idClickable = `id-${uniqueIdentifier.replace(/[.=\\]/g, '\\$&')}`;
```

Comment:
Hmm I think this is the only shared component so I think this is safe since it's just an id name that's being updated. @vbahinwillit to double check me since the original change was made in a [PR 3 years ago](https://github.com/department-of-veterans-affairs/vets-website/pull/22308).

Reply:
I agree with John. The id is used select appointments in the appointments list. So I just did a quick test to see if appointments could be selected via the mouse and keyboard to display the appointments detail view.

However, there are some other things happening. I am not sure if its related or not.

**Pending appointments is not wide enough**
![Screenshot 2025-05-01 at 4 14 14 PM](https://github.com/user-attachments/assets/85db7baa-968d-461b-bb9c-1bafebe13f28)

**Back to past appointments is missing on the back arrow**
![Screenshot 2025-05-01 at 4 19 59 PM](https://github.com/user-attachments/assets/1d1b0240-6fc1-467c-9cd5-f598d192306d)

Reply:
Yeah those seem unrelated to these changes. I'll check to see if we did anything to to cause those bugs though. The appointment list should be full width?

Reply:
@vbahinwillit I couldn't reproduce the back button text missing. But with the width of the pending appointments list if you add `usa-unstyled-list vaos-appts__list` to the classes for those `<ul>`s it allows the items to go full width.